controllers/members.js

```diff
@@ -85,8 +86,15 @@ const archiveMembers = async (req, res) => {
   try {
     const { username } = req.params;
     const user = await fetchUser({ username });
+
+    const userId = user.user.id;
+    const currentUserID = req.userData.id;
+    const body = req.body;
+    if (body?.reason === "" || body?.reason === undefined) {
```

Comment:
reason can be " " as well

Reply:
updated!, use regex to verify empty/ undefined reason